KornSW.SwaggerWcf.nuspec

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>KornSW.SwaggerWcf</id>
+    <version>1.2.16-localbuild</version>
+    <title>KornSW.SwaggerWcf (fork)</title>

```

Comment:
This changes should be excluded from merge

Reply:
you are right - sorry
ill create a new pull-request (was from wrong branch)